src/components/containers/create-container.js

```diff
@@ -47,7 +47,11 @@ const combineDefaultEvents = (defaultEvents) => {
     ([target, eventsArray]) => {
       return {
         target,
-        eventHandlers: combineEventHandlers(eventsArray.map((event) => event.eventHandlers))
```

Comment:
Do we want empty objects or just "only real event" objects?

A filter might be better, but I'm not familiar with this code:

```js
eventHandlers: combineEventHandlers(
  eventsArray
    // Remove empties
    .filter(Boolean)
    // Capture property
    .map((event) => event.eventHandlers)
)
```

Reply:
cc @boygirl ^^^^

Reply:
empty object should be fine here.

Reply:
But really... we shouldn't be returning anything for that target if there are no events. Just extra work. @ryota-murakami I'll add a more comprehensive check and get a patch out today.